README.md

```diff
@@ -1,31 +1,34 @@
 # EP-Launch3
 
-[![GitHub release](https://img.shields.io/github/release/nrel/ep-launch.svg)](https://github.com/nrel/ep-launch/releases/latest)
+[![GitHub release](https://img.shields.io/github/release/nrel/ep-launch.svg?style=for-the-badge)](https://github.com/nrel/ep-launch/releases/latest)
```

Comment:
Updated badge styles in the readme, and moved links from old test results to Github Actions.  Also updated commentary as needed.